modules/ROOT/pages/auradb/managing-databases/advanced-metrics.adoc

```diff
@@ -0,0 +1,66 @@
+[[aura-monitoring]]
+= Advanced metrics
+
+label:AuraDB-Professional[]
+label:AuraDB-Enterprise[]
+
+Advanced metrics is a feature that enables access to a broad range of different instance and database metrics.
+
+To access *Advanced metrics*:
+
+. Navigate to the https://console.neo4j.io/?product=aura-db[Neo4j Aura Console] in your browser.
+. Select the instance you want to access.
+. Select the *Metrics* tab.
+. Select the *Advanced metrics* button.
+
+The presented metrics will be laid out across three tabs according to their category:
+
+* *Resources* - Overall system resources, such as CPU, RAM and disk usage.
+* *Instance* - Information about the Neo4j instance(s) running the database.
+* *Database* - Metrics concerning the database itself, such as usage statistics and entity counts.
+
+When viewing metrics, you can select from the following time intervals:
+
+* 30 minutes
+* 6 hours
+* 24 hours
+* 3 days
+* 7 days
+* 14 days
+* 30 days
+
+== Chart options
+
+[NOTE]
+====
+Some charts can be toggled between absolute and relative values using the *%* toggle.
+====
+
+To access the chart options menu, select the *...* button on any chart.
+
+=== Reset zoom
+
+If the zoom level has been altered by selecting and dragging across a chart, *Reset zoom* resets the zoom back to the selected interval.
+
+=== Expand
+
+*Expand* gives you a better view of a particular metric by expanding the chart to take up all the available screen estate.
+
+=== More info
+
+Selecting *More info* brings up an explanation of the particular metric.
+
+== Aggregations
+
+label:AuraDB-Enterprise[]
+
+A database may be underpinned by more than one cluster member for increased resilience and performance.
+
+Metrics coming from several cluster members will be aggregated. To convey a more detailed picture of the state of the system, several aggregations can be shown.
+
+The possible aggregations are:
+
+* *Min* The minimum value of the metric across all cluster members.
+* *Max* The maximum value of the metric across all cluster members.
+* *Average* The average value of the metric across all cluster members.
+* *Sum* The summarized value of the metric across all cluster members.
```

Comment:
I am not sure 'summarized' is the right word here 🤔. Maybe 'The sum of the values of the metric across all cluster members'

Reply:
You just disapprove of the US spelling 😛